src/runtime/transformers/shiki/shiki.ts

```diff
@@ -8,39 +9,53 @@ export default defineTransformer({
   extensions: ['.md'],
   transform: async (content, options = {}) => {
     const shikiHighlighter = useShikiHighlighter(options)
-    const colorMap: TokenColorMap = {}
-    const codeBlocks: any[] = []
-    const inlineCodes: any = []
-    visit(
-      content.body,
-      (node: any) => (node.tag === 'code' && node?.props.code) || (node.tag === 'code-inline' && (node.props?.lang || node.props?.language)),
-      (node) => {
-        if (node.tag === 'code') {
-          codeBlocks.push(node)
-        } else if (node.tag === 'code-inline') {
-          inlineCodes.push(node)
-        }
-      }
-    )
 
-    await Promise.all(codeBlocks.map(highlightBlock))
-    await Promise.all(inlineCodes.map(highlightInline))
+    // TODO copy highlighted body to excerpt if it is the same
```

Comment:
```suggestion

```
Based on your other PR, https://github.com/nuxt/content/pull/1801#issuecomment-1373854525. I think we can simply remove this line

Reply:
Done